.travis.yml

```diff
@@ -0,0 +1,20 @@
+lenguage: node_js
+node_js: 
+  - "6.9"
+install:
+  - npm install
+cache:
+  directories:
+    - node_modules
+script:
+  - npm run build
+  - npm test
+deploy:
+  provider: npm
+  email: "pytechmike@gmail.com"
+  api_key: "7afce674150a3fcbe06118d34c6b3f03711983a9"
```

Comment:
Don't publish your keys as they can be stolen. Destroy this key and generate new. Use Environment Variables set in Travis project settings for `email` and `api_key` like so `api_key: "$NPM_TOKEN"`